Source/VMware.vSphereDSC/Tests/Unit/VMHostVss.Unit.Tests.ps1

```diff
@@ -712,4 +649,4 @@ try {
 finally {
     # Calls the function to Remove the mocked VMware.VimAutomation.Core module after all tests.
     Invoke-TestCleanup
-}
+}
```

Comment:
Add an empty line at the end of the file.

Reply:
Changed